include/miopen/miopen.h

```diff
@@ -6582,6 +6582,39 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+
+// GLU APIs
+/** @addtogroup glu
```

Comment:
Add * @defgroup glu at the beginning of miopen.h.

Reply:
I have added * @defgroup glu miopen.h at [66bd845](https://github.com/moreh-dev/MIOpen/pull/6/commits/66bd84579f508fe2d2deb6124c3c962f9bbc2a83)